src/Nethermind/Nethermind.Blockchain.Test/GenesisBuilderTests.cs

```diff
@@ -61,17 +61,15 @@ private Block GetGenesisBlock(string chainspecPath)
         specProvider.GetSpec(Arg.Any<ForkActivation>()).Returns(Berlin.Instance);
         ITransactionProcessor transactionProcessor = Substitute.For<ITransactionProcessor>();
         IGenesisPostProcessor genesisPostProcessor = Substitute.For<IGenesisPostProcessor>();
-        GenesisLoader genesisLoader = new(
+        GenesisBuilder genesisLoader = new(
             chainSpec,
             specProvider,
-            worldStateManager.GlobalStateReader,
             stateProvider,
             transactionProcessor,
-            genesisPostProcessor,
-            LimboLogs.Instance);
+            [genesisPostProcessor]);
```

Comment:
after params, you can do just:
```suggestion
            genesisPostProcessor);
```